src/inverseRobot.js

```diff
@@ -7,7 +7,21 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const copyRobot = {};
+
+  for (const key in robot) {
+    if (robot[key]) {
```

Comment:
actually, the same as before:
![image](https://github.com/mate-academy/js_inverse_robot/assets/86923089/087076e7-6be1-479f-9d2e-d141c851d66d)

this check is redundant, for example, if the value is 0 and the key is let's say `version` you will not add the property `'0': 'version'` because of this check (0 is falsy value)